<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickableEditDlg</class>
 <widget class="QDialog" name="PickableEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Stackable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fStackable">
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fAmount"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Uses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fUses">
     <property name="minimum">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Item slot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fItemSlot"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fCategory"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Armor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fArmor"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fWeight"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fPrice"/>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fTarget"/>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Damage</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fDamage"/>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Script ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fScript"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PickableEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PickableEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
